index.html

```diff
@@ -245,5 +243,94 @@ <h2>Skills</h2>
         </li>
       </ul>
     </section>
+    <!-- Part III -->
+    <section class="contact-form">
+      <h3 class="contact-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+
+      <form
+        class="contact-elements"
+        action="https://formspree.io/f/mlezlyew"
+        method="POST"
+      >
+        <input
+          class="contact-name celement"
+          type="text"
+          required
+          maxlength="30"
+          placeholder="Full name"
+        />
```

Comment:
- [ ] The form submission doesn't work as expected. I would recommend adding `name="name"` to the input of name to be able to submit the data to the Formspree.

<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/159586996-5f58e338-3328-48c6-af88-faa3d9dc48df.png">

Reply:
just noticed this a minute after i hit submit for review, but I corrected it now, thanks for the feedback